libr/anal/fcn.c

```diff
@@ -1209,14 +1261,42 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 			gotoBeachRet ();
 #endif
 			break;
+		case R_ANAL_OP_TYPE_CMP:
+			cmpval = op.ptr;
```

Comment:
I think this value is usually taken in a different way in x86 at least. Right @sivaramaa?